### PR TITLE
Update the Quickstart Guide

### PR DESCRIPTION
**Pull request by Morph1984**
- Replaced Kosmos with Atmosphere
- Added save file dumping section
- Provided more detail in some sections
- Streamlined the dumping process

### CODE REVIEW

**Comment by Eloeri** [comment]
lgtm

**Comment by goldenx86** [comment]
I like it too.

File: site/content/help/quickstart/index.md [hunk 1]
```diff
@@ -76,61 +91,87 @@ This process should take about 60 to 90 minutes.
...
-    - 5f. Either use the touchscreen or navigate using your controller, and choose `nxdumptool`
-    - 5g. Chose the `Dump gamecard content` option. 
-    - 5h. Chose the `Cartridge Image (XCI) dump` option. 
-    - 5i. Make sure to dump the certificate with the cartridge. The options can be toggled by pressing `left` or `right`. 
-    - 5j. Once the cartridge image has been dumped, place it into a new folder in a location of your choice.
+
+# Dumping Cartridge Games
+
+6. We will now dump the `Cartridge Image (XCI)` files from your game cartridge, to use in yuzu. Insert the game cartridge of your choice.
+    - 6a. Boot your Nintendo Switch into RCM mode (steps 2d to 2f), and make sure it is connected to your computer.
```
Comment (CaptV0rt3x) [inline site/content/help/quickstart/index.md]: Unnecessary comma

File: site/content/help/quickstart/index.md [hunk 2]
```diff
@@ -76,61 +91,87 @@ This process should take about 60 to 90 minutes.
...
+    - 6b. Run TegraRcmGUI, and in the `Payload` tab of TegraRcmGUI, click on the folder icon and navigate to the `hekate.bin` file you extracted earlier. (step 3b)
```
Comment (CaptV0rt3x) [inline site/content/help/quickstart/index.md]: Ditto

File: site/content/help/quickstart/index.md [hunk 3]
```diff
@@ -76,61 +91,87 @@ This process should take about 60 to 90 minutes.
...
+    - 6k. Once the cartridge image has been dumped, power off your switch by holding the power button for a few seconds, then select the Power Options > Turn off.
+    - 6l. With your Switch powered off, remove the SD card.
+    - 6m. Insert your SD card into your computer. If the XCI was dumped in parts combine them with this command, for example: `copy /b "Super Mario Odyssey.xc0" + "Super Mario Odyssey.xc1" "Super Mario Odyssey.xci"`. Now you can place this XCI in a game directory of your choice.
+
+# Dumping Save Files (Optional)
+
+7. We will now dump the games' save files from your switch to use in yuzu.
+    - 7a. Download [Checkpoint.nro](https://github.com/FlagBrew/Checkpoint/releases)
+    - 7b. Insert your SD card into your computer.
+    - 7c. Create a folder named `Checkpoint` within the `switch` folder of your SD card, and place the `Checkpoint.nro` file into the `Checkpoint` folder you have just created.
```
Comment (CaptV0rt3x) [inline site/content/help/quickstart/index.md]: Same as `nxdumptool` - repetitive

File: site/content/help/quickstart/index.md [hunk 4]
```diff
@@ -76,61 +91,87 @@ This process should take about 60 to 90 minutes.
...
+    - 4f. Power off your switch to avoid damage to your SD card by selecting `Back` and then selecting `Power Off` and then eject your SD card. Insert your SD card into your computer and use your file explorer to open the contents. Place the `yuzu_copy.bat` file you downloaded earlier into the root of your SD card. Double-click it to start combining your `rawnand.bin` if it was dumped into sections i.e. `rawnand.bin.00`. If your rawnand.bin dumped as one ~30gb file, then proceed with the guide. 
+    - 4g. If you needed to dump the rawnand.bin into sections due to lack of space on your sd card, transfer the files that did dump to a new folder on your desktop, and then resume the backup of your `eMMC RAW GPP`. After you have obtained all of the `rawnand.bin` files, add them to the same folder as before, place the `yuzu_copy.bat` file you downloaded earlier into that same folder and double-click to run. After they are combined, please proceed with the guide.
+
```
[the remark this answers is not in the document]
Comment (Morph1984) [reply site/content/help/quickstart/index.md]: This was basically unmodified from the initial guide.

[the remark this answers is not in the document]
Comment (Morph1984) [reply site/content/help/quickstart/index.md]: > Steps 4d - 4g look lengthy. Need to see render.

These steps were unmodified from the initial guide

File: site/content/help/quickstart/index.md [hunk 5]
```diff
@@ -24,45 +25,59 @@ Sample Image:
...
+    - For Windows, this is '%APPDATA%\yuzu' or 'C:\Users\{username}\AppData\Roaming\yuzu'
+    - For Linux, this is '~/.local/share/yuzu'
+
+
+# Preparing the microSD Card
+
+1. We will now prepare the microSD card.
+    - 1a. Extract the contents of the `atmosphere` and `hekate` zip files into the root of your SD card.
+    - 1b. Place the `fusee-primary.bin` and `Lockpick_RCM.bin` files into the `bootloader/payloads` folder of the SD card.
+    - 1c. Create a folder named `nxdumptool` within the `switch` folder of your SD card, and place the `nxdumptool.nro` file into the `nxdumptool` folder you have just created.
```
[the remark this answers is not in the document]
Comment (Morph1984) [reply site/content/help/quickstart/index.md]: Addressed.

File: site/content/help/quickstart/index.md [hunk 6]
```diff
@@ -24,45 +25,59 @@ Sample Image:
...
-To see if your Switch is hackable, visit https://damota.me/ssnc/checker and test your Switch's serial number. 
+To start playing games, yuzu needs a couple of system files and folders from your switch in order to play them properly.
+To see if your Switch is hackable, visit https://damota.me/ssnc/checker and test your Switch's serial number.
 If your Switch is patched, you will be unable to complete the following steps.
+The Switch v2 (Mariko) and the Switch Lite are both patched and you will not be able to complete the following steps.
 
 This guide will help you copy all your system files, games, updates, and DLC from your switch to your computer and organize them in a format yuzu understands.
-Make sure to place your Nintendo Switch into Airplane mode before starting this guide.
+
+**IMPORTANT: Make sure to place your Nintendo Switch into Airplane Mode before starting this guide.** System Settings -> Airplane Mode -> Airplane Mode **ON**
```
[the remark this answers is not in the document]
Comment (Morph1984) [reply site/content/help/quickstart/index.md]: Addressed.

File: site/content/help/quickstart/index.md [hunk 7]
```diff
@@ -76,61 +91,87 @@ This process should take about 60 to 90 minutes.
...
-    - 2e. Click on `Lockpick_RCM.bin` in the list of payloads. 
-    - 2f. After Lockpick_RCM has successfully booted, Press the power button to select `Dump from SysNAND`. 
-    - 2g. It will prompt you to press `VOL+` or `VOL-` to reboot to sept. Do as it says and it will start dumping process. 
-    - 2h. After Lockpick_RCM has finished deriving the keys, please make note of the location of the key file. Default is: `sd:/switch/prod.keys` and `sd:/switch/title.keys`.
-    - 2i. With both `prod.keys` and `title.keys` dumped from your Nintendo Switch, place them in the `%YUZU_DIR%/keys` directory. 
+
+# Dumping Prod.keys and Title.keys
+
+3. We will now dump your `prod.keys` and `title.keys` for decryption of your game files.
+    - 3a. Boot your Nintendo Switch into RCM mode (steps 2d to 2f), and make sure it is connected to your computer.
```
[the remark this answers is not in the document]
Comment (Morph1984) [reply site/content/help/quickstart/index.md]: Addressed.

File: site/content/help/quickstart/index.md [hunk 8]
```diff
@@ -76,61 +91,87 @@ This process should take about 60 to 90 minutes.
...
+    - 3b. Extract the `hekate.bin` file from the `hekate` zip file you downloaded from the prerequisites.
+    - 3c. Run TegraRcmGUI, and in the `Payload` tab of TegraRcmGUI, click on the folder icon and navigate to the `hekate.bin` file you extracted earlier.
```
[the remark this answers is not in the document]
Comment (Morph1984) [reply site/content/help/quickstart/index.md]: Addressed.

File: site/content/help/quickstart/index.md [hunk 9]
```diff
@@ -76,61 +91,87 @@ This process should take about 60 to 90 minutes.
...
+    - 3d. Click on `Inject Payload`, and your Switch will boot into the Hekate menu.
+    - 3e. When it has successfully booted into the Hekate menu, tap on `Payloads`. This will show a list of payloads.
+    - 3f. Tap on `Lockpick_RCM.bin` in the list of payloads.
+    - 3g. After Lockpick_RCM has successfully booted, press the power button to select `Dump from SysNAND`. 
+    - 3h. It will automatically boot to sept and start deriving the keys. Wait for it to finish deriving the keys.
+    - 3i. After Lockpick_RCM has finished deriving the keys, please make note of the location of the key files. Default is: `sd:/switch/prod.keys` and `sd:/switch/title.keys`.
+    - 3j. Press any key to return to the menu, then navigate with the VOL+/VOL- buttons to highlight and select `Power Off` by pressing the power button.
+    - 3h. With your Switch powered off, remove the SD card.
```
[the remark this answers is not in the document]
Comment (Morph1984) [reply site/content/help/quickstart/index.md]: Addressed.

File: site/content/help/quickstart/index.md [hunk 10]
```diff
@@ -76,61 +91,87 @@ This process should take about 60 to 90 minutes.
...
-    - 4f. Double-click on the row named `USER`. Enter the same key. Click `Mount`, and the tool may become unresponsive again. Once this mounts, open the new drive and copy the `Contents` and `save` folders to `%YUZU_DIR%/nand/user`. After, click `Unmount` and close the window and then the program.
-    - 4g. If you have games or game data stored on your sd card, copy the `Nintendo` folder of your sd card into `%YUZU_DIR%/sdmc`.
+
+4. We will now boot Hekate to dump your system files:
+    - 4a. Boot your Nintendo Switch into RCM mode (steps 2d to 2f), and make sure it is connected to your computer.
```
[the remark this answers is not in the document]
Comment (Morph1984) [reply site/content/help/quickstart/index.md]: Addressed.

File: site/content/help/quickstart/index.md [hunk 11]
```diff
@@ -76,61 +91,87 @@ This process should take about 60 to 90 minutes.
...
+    - 4b. Run TegraRcmGUI, and in the `Payload` tab of TegraRcmGUI, click on the folder icon and navigate to the `hekate.bin` file you extracted earlier. (step 3b)
```
[the remark this answers is not in the document]
Comment (Morph1984) [reply site/content/help/quickstart/index.md]: Addressed.

File: site/content/help/quickstart/index.md [hunk 12]
```diff
@@ -76,61 +91,87 @@ This process should take about 60 to 90 minutes.
...
-    - 5e. In the Hekate browser on your Nintendo Switch, click on `Launch`, and then click on `CFW (SYSNAND)`. Your Switch will launch into Custom Firmware Mode (CFW), and once your Switch has booted into the home menu, while holding the `R` button, click on the Album application. This will launch the Homebrew Menu.
-    - 5f. Either use the touchscreen or navigate using your controller, and choose `nxdumptool`
-    - 5g. Chose the `Dump gamecard content` option. 
-    - 5h. Chose the `Cartridge Image (XCI) dump` option. 
-    - 5i. Make sure to dump the certificate with the cartridge. The options can be toggled by pressing `left` or `right`. 
-    - 5j. Once the cartridge image has been dumped, place it into a new folder in a location of your choice.
+
+# Dumping Cartridge Games
+
+6. We will now dump the `Cartridge Image (XCI)` files from your game cartridge, to use in yuzu. Insert the game cartridge of your choice.
```
[the remark this answers is not in the document]
Comment (Morph1984) [reply site/content/help/quickstart/index.md]: Addressed.

File: site/content/help/quickstart/index.md (hunk 1 above)
Comment (Morph1984) [reply site/content/help/quickstart/index.md]: Addressed.

File: site/content/help/quickstart/index.md (hunk 2 above)
Comment (Morph1984) [reply site/content/help/quickstart/index.md]: Addressed.

File: site/content/help/quickstart/index.md (hunk 3 above)
Comment (Morph1984) [reply site/content/help/quickstart/index.md]: Addressed.

File: site/content/help/quickstart/index.md [hunk 13]
```diff
@@ -24,45 +25,59 @@ Sample Image:
 
 # yuzu Quickstart Guide
 
-To start playing games, yuzu needs a couple of different files and directories from your switch in order to play them properly. 
-To see if your Switch is hackable, visit https://damota.me/ssnc/checker and test your Switch's serial number. 
+To start playing games, yuzu needs a couple of system files and folders from your switch in order to play them properly.
+To see if your Switch is hackable, visit https://damota.me/ssnc/checker and test your Switch's serial number.
```
[the remark this answers is not in the document]
Comment (Morph1984) [reply site/content/help/quickstart/index.md]: Addressed.

File: site/content/help/quickstart/index.md [hunk 14]
```diff
@@ -24,45 +25,59 @@ Sample Image:
...
 If your Switch is patched, you will be unable to complete the following steps.
```
[the remark this answers is not in the document]
Comment (Morph1984) [reply site/content/help/quickstart/index.md]: Addressed.

File: site/content/help/quickstart/index.md [hunk 15]
```diff
@@ -24,45 +25,59 @@ Sample Image:
...
+The Switch v2 (Mariko) and the Switch Lite are both patched and you will not be able to complete the following steps.
 
 This guide will help you copy all your system files, games, updates, and DLC from your switch to your computer and organize them in a format yuzu understands.
-Make sure to place your Nintendo Switch into Airplane mode before starting this guide.
+
+**IMPORTANT: Make sure to place your Nintendo Switch into Airplane Mode before starting this guide.** System Settings -> Airplane Mode -> Airplane Mode **ON**
+
 This process should take about 60 to 90 minutes.
```
[the remark this answers is not in the document]
Comment (Morph1984) [reply site/content/help/quickstart/index.md]: Addressed.

File: site/content/help/quickstart/index.md [hunk 16]
```diff
@@ -24,45 +25,59 @@ Sample Image:
...
+
+
+# Preparing the microSD Card
+
+1. We will now prepare the microSD card.
+    - 1a. Extract the contents of the `atmosphere` and `hekate` zip files into the root of your SD card.
+    - 1b. Place the `fusee-primary.bin` and `Lockpick_RCM.bin` files into the `bootloader/payloads` folder of the SD card.
+    - 1c. Create a folder named `nxdumptool` within the `switch` folder of your SD card, and place the `nxdumptool.nro` file into the `nxdumptool` folder you have just created.
+    - 1d. Once done, eject the SD card and place it into your Nintendo Switch.
+        ![SD_Template](./sd_template.png)
```
[the remark this answers is not in the document]
Comment (Morph1984) [reply site/content/help/quickstart/index.md]: Addressed.

File: site/content/help/quickstart/index.md [hunk 17]
```diff
@@ -24,45 +25,59 @@ Sample Image:
...
+
+# Dumping BIS Keys
+
+2. We will now dump your BIS (Built-In Storage) keys for use in later decryption.
+    - 2a. Run the TegraRcmGUI installer you downloaded from the prerequisites, and after installation, start the program. 
+    - 2b. In the `Settings` tab, click on `Install Driver` which will install the drivers necessary for your computer to interface with your Nintendo Switch. 
+    - 2c. After the drivers have been installed, plug your Nintendo Switch into your computer.
+    - 2d. Power off your Switch while it is still connected to your computer.
+    - 2e. Insert your RCM jig into the right joy-con slot, make sure it is seated securely at the base, and then press VOL+ and Power buttons at the same time. Nothing should happen on your Switch; if the switch starts to turn on normally, go back to the beginning of step 1d and try again.
+    - 2f. In the program TegraRcmGUI you should see the Nintendo Switch icon in the lower left corner flash green and state `RCM O.K.`
```
[the remark this answers is not in the document]
Comment (Morph1984) [reply site/content/help/quickstart/index.md]: Addressed.

File: site/content/help/quickstart/index.md [hunk 18]
```diff
@@ -24,45 +25,59 @@ Sample Image:
...
+    - 2g. In the `Tools` tab of TegraRcmGUI, click on `biskeydump (by rajkosto)`.
+    - 2h. Your Switch will briefly flash with a blue QR code, you may or may not see this, and your computer will prompt you to save a text document titled `BIS_keys.txt`. Navigate to `%YUZU_DIR%/keys` and save the file as `console.keys`. Open the file to make sure you have the following key entries.
```
[the remark this answers is not in the document]
Comment (Morph1984) [reply site/content/help/quickstart/index.md]: Addressed.

File: site/content/help/quickstart/index.md [hunk 19]
```diff
@@ -76,61 +91,87 @@ This process should take about 60 to 90 minutes.
...
-    - 2h. After Lockpick_RCM has finished deriving the keys, please make note of the location of the key file. Default is: `sd:/switch/prod.keys` and `sd:/switch/title.keys`.
-    - 2i. With both `prod.keys` and `title.keys` dumped from your Nintendo Switch, place them in the `%YUZU_DIR%/keys` directory. 
+
+# Dumping Prod.keys and Title.keys
+
+3. We will now dump your `prod.keys` and `title.keys` for decryption of your game files.
+    - 3a. Boot your Nintendo Switch into RCM mode (steps 2d to 2f), and make sure it is connected to your computer.
+    - 3b. Extract the `hekate.bin` file from the `hekate` zip file you downloaded from the prerequisites.
+    - 3c. Run TegraRcmGUI, and in the `Payload` tab of TegraRcmGUI, click on the folder icon and navigate to the `hekate.bin` file you extracted earlier.
+    - 3d. Click on `Inject Payload`, and your Switch will boot into the Hekate menu.
```
[the remark this answers is not in the document]
Comment (Morph1984) [reply site/content/help/quickstart/index.md]: Addressed.

File: site/content/help/quickstart/index.md [hunk 20]
```diff
@@ -76,61 +91,87 @@ This process should take about 60 to 90 minutes.
...
-    - 4f. Double-click on the row named `USER`. Enter the same key. Click `Mount`, and the tool may become unresponsive again. Once this mounts, open the new drive and copy the `Contents` and `save` folders to `%YUZU_DIR%/nand/user`. After, click `Unmount` and close the window and then the program.
-    - 4g. If you have games or game data stored on your sd card, copy the `Nintendo` folder of your sd card into `%YUZU_DIR%/sdmc`.
+
+4. We will now boot Hekate to dump your system files:
+    - 4a. Boot your Nintendo Switch into RCM mode (steps 2d to 2f), and make sure it is connected to your computer.
+    - 4b. Run TegraRcmGUI, and in the `Payload` tab of TegraRcmGUI, click on the folder icon and navigate to the `hekate.bin` file you extracted earlier. (step 3b)
+    - 4c. Click on `Inject Payload`, and your Switch will boot into the Hekate menu.
```
[the remark this answers is not in the document]
Comment (Morph1984) [reply site/content/help/quickstart/index.md]: Addressed.

File: site/content/help/quickstart/index.md [hunk 21]
```diff
@@ -76,61 +91,87 @@ This process should take about 60 to 90 minutes.
...
+    - 4f. Power off your switch to avoid damage to your SD card by selecting `Back` and then selecting `Power Off` and then eject your SD card. Insert your SD card into your computer and use your file explorer to open the contents. Place the `yuzu_copy.bat` file you downloaded earlier into the root of your SD card. Double-click it to start combining your `rawnand.bin` if it was dumped into sections i.e. `rawnand.bin.00`. If your rawnand.bin dumped as one ~30gb file, then proceed with the guide. 
+    - 4g. If you needed to dump the rawnand.bin into sections due to lack of space on your sd card, transfer the files that did dump to a new folder on your desktop, and then resume the backup of your `eMMC RAW GPP`. After you have obtained all of the `rawnand.bin` files, add them to the same folder as before, place the `yuzu_copy.bat` file you downloaded earlier into that same folder and double-click to run. After they are combined, please proceed with the guide.
+
+# Moving SYSTEM and USER to the yuzu directory
+
+5. We will now extract the necessary directories from your rawnand.bin for yuzu to read your system files.
```
[the remark this answers is not in the document]
Comment (Morph1984) [reply site/content/help/quickstart/index.md]: Addressed.

File: site/content/help/quickstart/index.md [hunk 22]
```diff
@@ -76,61 +91,87 @@ This process should take about 60 to 90 minutes.
...
+    - 5d. Copy the `PRODINFO.bin` and `BCPKG2-1-Normal-Main.bin` files to the `%YUZU_DIR%/sysdata` directory.
+    - 5e. Double-click on the row named `SYSTEM`. In the keys box, enter the `bis_key2_crypt` key in the crypt box and the `bis_key2_tweak` key in the tweak box and click test. If the text is red, double check your keys. If the text is blue, click on `Install` and wait until it says `Driver (ver XXXXX) installed, Service is running` where XXXXX is anything, then select `Y:` in the dropdown and click `Mount`. The application may become unresponsive for a moment. Press Windows-R and type `Y:` into the box and click OK and there will be a couple of folders with some named `Contents` and `save`. Copy all of these into `%YUZU_DIR%/nand/system`. After this is done, click `Unmount` and close the window.
+    - 5f. Double-click on the row named `USER`. Enter the same key. Click `Mount`, and the tool may become unresponsive again. Once this mounts, open the new drive and copy the `Contents` and `save` folders to `%YUZU_DIR%/nand/user`. After, click `Unmount` and close the window and then the program.
```
[the remark this answers is not in the document]
Comment (Morph1984) [reply site/content/help/quickstart/index.md]: Addressed.

**Pull request merged**